mzroll/mainwindow.cpp

```diff
@@ -3093,6 +2946,245 @@ QWidget* MainWindow::eicWidgetController() {
+				SLOT(startSpectralAveraging()));
+
+		return btnAverageSpectra;
+
+	}
+	else if (btnName == "btnLast") {
+
+		QToolButton *btnLast = new QToolButton(parent);
+		btnLast->setIcon(QIcon(rsrcPath + "/last.png"));
+		btnLast->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnLast->setToolTip(tr("History Back (Ctrl+Left)"));
+		btnLast->setShortcut(tr("Ctrl+Left"));
+		connect(btnLast, SIGNAL(clicked()), mw, SLOT(historyLast()));
+		return btnLast;
+
+	}
+	else if (btnName == "btnNext") {
+
+		QToolButton *btnNext = new QToolButton(parent);
+		btnNext->setIcon(QIcon(rsrcPath + "/next.png"));
+		btnNext->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnNext->setToolTip(tr("History Forward (Ctrl+Right)"));
+		btnNext->setShortcut(tr("Ctrl+Right"));
+		connect(btnNext, SIGNAL(clicked()), mw, SLOT(historyNext()));		
+		return btnNext;
+
+	}
+	else if (btnName == "btnPDF") {
+
+		QToolButton *btnPDF = new QToolButton(parent);
+		btnPDF->setIcon(QIcon(rsrcPath + "/exportpdf.png"));
+		btnPDF->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnPDF->setToolTip(tr("Save EIC Image to PDF file"));
+		connect(btnPDF, SIGNAL(clicked()), mw, SLOT(exportPDF()));
+		return btnPDF;
+
+	}
+	else if (btnName == "btnPNG") {
+
+		QToolButton *btnPNG = new QToolButton(parent);
+		btnPNG->setIcon(QIcon(rsrcPath + "/copyPNG.png"));
+		btnPNG->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnPNG->setToolTip(tr("Copy EIC Image to Clipboard"));
+		connect(btnPNG, SIGNAL(clicked()), mw, SLOT(exportSVG()));
+		return btnPNG;
+
+	}
+	else if (btnName == "btnPrint") {
+
+		QToolButton *btnPrint = new QToolButton(parent);
+		btnPrint->setIcon(QIcon(rsrcPath + "/fileprint.png"));
+		btnPrint->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnPrint->setToolTip(tr("Print EIC (Ctr+P)"));
+		btnPrint->setShortcut(tr("Ctrl+P"));
+		connect(btnPrint, SIGNAL(clicked()), mw, SLOT(print()));
+		return btnPrint;
+
+	}
+	else if (btnName == "btnAutoZoom") {
+
+		QToolButton *btnAutoZoom = new QToolButton(parent);
+		btnAutoZoom->setCheckable(true);
+		btnAutoZoom->setChecked(true);
+		btnAutoZoom->setIcon(QIcon(rsrcPath + "/autofocus.png"));
+		btnAutoZoom->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnAutoZoom->setToolTip(
+			tr("Auto Zoom. Always center chromatogram on expected retention time!"));
+
+		connect(btnAutoZoom, SIGNAL(toggled(bool)), mw->getEicWidget(),
+				SLOT(autoZoom(bool)));
+
+		return btnAutoZoom;
+	}
+	else if (btnName == "btnShowTic") {
+
+		QToolButton *btnShowTic = new QToolButton(parent);
+		btnShowTic->setCheckable(true);
+		btnShowTic->setChecked(false);
+		btnShowTic->setIcon(QIcon(rsrcPath + "/tic.png"));
+		btnShowTic->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+		btnShowTic->setToolTip(tr("Show TICs"));
+		connect(btnShowTic, SIGNAL(toggled(bool)), mw->getEicWidget(), SLOT(replot()));
+		connect(btnShowTic, SIGNAL(toggled(bool)), mw->getEicWidget(),
+				SLOT(showTicLine(bool)));
+
```

Comment:
The behaviour of this button is changed as compared with the same button of develop branch. Have a look.